services/frontend-service/src/ui/components/CommitInfo/CommitInfo.tsx

```diff
@@ -61,7 +61,48 @@ export const CommitInfo: React.FC<CommitInfoProps> = (props) => {
                         </tr>
                     </tbody>
                 </table>
+                <h2>Events</h2>
+                <CommitInfoEvents events={commitInfo.events} />
             </main>
         </div>
     );
 };
+
+const CommitInfoEvents: React.FC<{ events: Event[] }> = (props) => (
+    <table border={1}>
+        <thead>
+            <tr>
+                <th>Date:</th>
+                <th>Event Description:</th>
+                <th>Environments:</th>
+            </tr>
+        </thead>
+        <tbody>
+            {props.events.map((event) => {
+                const createdAt = event.createdAt?.toISOString() || '';
+                const [description, environments] = eventDescription(event);
+                return (
+                    <tr>
+                        <td>{createdAt}</td>
+                        <td>{description}</td>
+                        <td>{environments.join(', ')}</td>
+                    </tr>
+                );
+            })}
+        </tbody>
+    </table>
+);
+
+const eventDescription = (event: Event): [string, string[]] => {
```

Comment:
I think in the long run we need to return a JSXElement here (instead of the first string), because there should be more formatting that just "stirng". However, for this PR I'm fine with how it is.